src/app/assets/lang/locale-en.json

```diff
@@ -57,6 +57,16 @@
 	"terraNoteEditor": {
 		"insertText": "Insert text."
 	},
+	"terraCodeEditor":
+	{
+		"changeViewOverlay":
+		{
+			"title":"Unknown code formatting",
+			"text":"Switching to Editor-View will cause unknown markup formatting to be lost. Are you sure you want to change the view ?",
```

Comment:
Ich würde dazu tendieren aus "Editor-View" in der englischen Version "editor view" zu machen. Es sei denn, ihr habt das bewusst als Eigennamen für die Ansicht gewählt.